From ec7afc2ceecaeb88ac45cd07e780bc8325c470d1 Mon Sep 17 00:00:00 2001
From: Tim S <stahta01@users.sourceforge.net>
Date: Sun, 14 May 2017 11:46:05 -0400
Subject: [PATCH 2/3] * mingw-w64-crt-git: Applying
 "uuid.c-Commented-out-DEFINE_GUID-also-in-extras-uuid.patch".

---
 mingw-w64-crt-git/PKGBUILD | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/mingw-w64-crt-git/PKGBUILD b/mingw-w64-crt-git/PKGBUILD
index 325bf357..adce3881 100644
--- a/mingw-w64-crt-git/PKGBUILD
+++ b/mingw-w64-crt-git/PKGBUILD
@@ -21,11 +21,13 @@ options=('!strip' 'staticlibs' '!buildflags' '!emptydirs')
 source=("mingw-w64"::"git+https://git.code.sf.net/p/mingw-w64/mingw-w64"
         "0003-dxgi-Add-missing-dxgi-1.2-structs-and-interfaces.patch"
         "0004-d3d11-Add-missing-d3d11-1.1-structs-and-interfaces.patch"
-        "0005-handle-ctor_list-internally.patch")
+        "0005-handle-ctor_list-internally.patch"
+        "uuid.c-Commented-out-DEFINE_GUID-also-in-extras-uuid.patch")
 sha256sums=('SKIP'
             '677642bbb0df3f1e0467f075f24c720657e9ed912a6e69cbf189352271fc09c1'
             'c6c996b273ee049a2c0b9570f007d2d76a95ff56abb3b147f156e6a221a561ef'
-            '0eabe00375d4e26fd7e9afa56f6a9da02845fe2c0f900f5e1f829842bff8624a')
+            '0eabe00375d4e26fd7e9afa56f6a9da02845fe2c0f900f5e1f829842bff8624a'
+            '949ee3ec369a7e69894281655fac6df283774961c3fad2a5929692f824714589')
 
 pkgver() {
   cd "${srcdir}/mingw-w64"
@@ -39,6 +41,7 @@ prepare() {
   cd ${srcdir}/mingw-w64
   git am --committer-date-is-author-date "${srcdir}"/0003-dxgi-Add-missing-dxgi-1.2-structs-and-interfaces.patch
   git am --committer-date-is-author-date "${srcdir}"/0004-d3d11-Add-missing-d3d11-1.1-structs-and-interfaces.patch
+  git apply                              "${srcdir}"/uuid.c-Commented-out-DEFINE_GUID-also-in-extras-uuid.patch
 }
 
 build() {
-- 
2.14.1.windows.1

